### ISSUE TITLE
Functional Test - Symfony\Bridge\Twig\Extension\CsrfExtension not found

### ISSUE THREAD

**Issue by silverbackdan** [symfony/symfony#26815]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | yes
| RFC?             | no
| Symfony version  | dev-master

While running CI on dev-master I get
```
Error: Class 'Symfony\Bridge\Twig\Extension\CsrfExtension' not found
```
In a functional test extending `WebTestCase` when calling `request` on the client.

It is working fine on 4.0. Here is my CI results:
https://travis-ci.org/silverbackis/ApiComponentBundle/builds/362645681?utm_source=email&utm_medium=notification
It has been this way for the last 7 days it seems.

I've had a quick look to see if perhaps I need to change my config for the test bundle or if Twig extensions now need to be installed separately but couldn't see anything. In the test form I have disabled csrf protection as well.

**Comment by xabbuh** [comment]
Status: Reviewed

**Comment by xabbuh** [comment]
will be fixed by #26818

Thanks for reporting it @silverbackdan.
